### PR TITLE
Kick off access work

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

File: extensions/amp-access/0.1/amp-access.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+ */
+
+import {assertHttpsUrl} from '../../../src/url';
+import {getService} from '../../../src/service';
+import {installStyles} from '../../../src/styles';
+import {isExperimentOn} from '../../../src/experiments';
+import {log} from '../../../src/log';
+
+
+/*eslint no-unused-vars: 0*/
```
Comment (erwinmombay) [inline extensions/amp-access/0.1/amp-access.js]: is this for the `let typdef` ? if so we can add a `Def` suffix and it should be ignored.

File: extensions/amp-access/0.1/amp-access.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+}
+
+
+/**
+ * @param {!Window} win
+ * @return {!AccessService}
+ */
+export function installAccessService(win) {
+  return getService(win, 'access', () => {
+    return new AccessService(win).start_();
```
Comment (erwinmombay) [inline extensions/amp-access/0.1/amp-access.js]: can you explain what the instances are when we wouldn't call `start`?

Comment (erwinmombay) [inline extensions/amp-access/0.1/amp-access.js]: (im guessing it is just for testing purposes?)

File: extensions/amp-access/0.1/amp-access.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+   */
+  buildConfig_() {
+    let config;
+    try {
+      config = JSON.parse(this.accessElement_.textContent);
+    } catch (e) {
+      throw new Error('Failed to parse "amp-access" JSON: ' + e);
+    }
+    return {
+      authorization: assertHttpsUrl(assert(config['authorization'],
```
Comment (erwinmombay) [inline extensions/amp-access/0.1/amp-access.js]: can you add the 2nd arg to assertHttpsUrl (element context)

File: extensions/amp-access/0.1/amp-access.js (hunk 1 above)
Comment (dvoytenko) [inline extensions/amp-access/0.1/amp-access.js]: Cool. Done.

File: extensions/amp-access/0.1/amp-access.js (hunk 2 above)
Comment (dvoytenko) [inline extensions/amp-access/0.1/amp-access.js]: Yeah, testing. But I also prefer to not do heavy calls in the constructor itself.

File: extensions/amp-access/0.1/amp-access.js (hunk 3 above)
Comment (dvoytenko) [inline extensions/amp-access/0.1/amp-access.js]: At this time we don't have an element context yet. Right now, `accessElement` is a `script` element  with `application/json` type. Thus adding error styling to it won't work.

**Comment by erwinmombay** [comment]
@dvoytenko LGTM

**Pull request merged**